app/controllers/application_controller.rb

```diff
@@ -296,7 +296,7 @@ def saved_redirect_url_or(default)
 
   def generic_exception(exception)
     logger.warn "Operation FAILED: #{exception}"
-    logger.debug exception.backtrace.join("\n")
+    logger.error exception.backtrace.join("\n")
```

Comment:
While I'm inclined to agree it's a problematic policy, having backtraces at debug level, we have a lot of this throughout the codebase.  If it's to be changed, it would have to be everywhere.

It should also be at the same level as the exception message itself, else it would look rather strange if the log level was set at `error`.